### ISSUE TITLE
LensFlare rotation bug and fix

### ISSUE THREAD

**Issue by coldi** [mrdoob/three.js#1124]
i think i found a little bug in LensFlare.js
when you first look at it in the scene it starts spinning around like crazy for a short time before it meets wantedRotation. this effect can be fixed if you set "flare.rotation = 0;" at the beginning of the rotation calculations on line 74:

```javascript
flare.rotation = 0; // fix
flare.wantedRotation = flare.x * Math.PI * 0.35; // multiplier increased from 0.25 to 0.35 to get same amount of rotation movement as intended
flare.rotation += ( flare.wantedRotation - flare.rotation ) * 0.35;
```

ofcourse everyone can easy workaround if you set customUpdateCallback and fix it in an own update function... just wanted to let you guys know and for future releases :)

**Comment by arya-s** [comment]
Why don't you make a pull request? Seem's like a perfect little contribution, if it really fixes it.

**Comment by dubejf** [comment]
Fixed in #6755. LensFlare rotation is initialized at 0. Closing.
